### DISCUSSION TITLE
Review

### DISCUSSION THREAD

**Discussion by FennyWilriani** [lab-brussels-1/all-about-trees-2#23] [Q&A]
Hello coaches, There are some of our issues that are ready to review on our project board. Please help us give your feedback and input on how are we doing on the project. 

- Help us think things through?
As there are quite some steps that need to be done and folders that are needed to open, we might miss some steps and misunderstood some instructions. please help us through the things that needed to be added or changed on the issues on `ready to review` board 

- Could you give us a reality check?
Whether we make good progress towards the project results, or if there are any suggestions that we have to do to improve our work as a team to achieve better results.

**Comment by sammou00** [comment]
Review : 

1- Well done very good job. We all are proud of you.

2- 
![image](https://user-images.githubusercontent.com/67711948/159980800-c7de36df-8b12-496e-89e6-0f4e7d7f4847.png)
No JavaScript skills ? well you don't need JavaScript for this project.
Limited skills of members ? you don't need a lot of skills too , we are here to learn and the project is simple.

**Reply by sammou00** [reply]
![image](https://user-images.githubusercontent.com/67711948/159981332-4594c9da-4f5c-4092-8921-45f3a9acbfec.png)

3 - Nice design, but where are the pictures ?

**Reply by sammou00** [reply]
4- Nice User-personas

**Reply by sammou00** [reply]
5 - Very nice development-strategy, It would have been nice if you have some pictures for tress.  because you have them in the all-about-trees repo
